### DISCUSSION TITLE
What advantages has LGT8F328-D ?

### DISCUSSION THREAD

**Discussion by SuperUserNameMan** [dbuezas/lgt8fx#141] [General]
Hello.

I clearly see the advantages of LGT8F328-P compared to ATmega328p.

But why would one want a LGT8F328-D ?

**Comment by jayzakk** [comment]
Price? 👀

**Comment by dwillmore** [comment]
From my poking about on AE, prices aren't very good.

**Comment by LaZsolt** [comment]
328D has 2 DACs  and 2 comparators. But ADC and internal voltage references much worse.
Not recommended for new designs.
